### PR TITLE
Refactoring input loading

### PR DESCRIPTION
**Pull request by lecorguille**
- Change the UI by removing the select of the method zip or single
- Simplify the main R script by creating function as initialized in Probmetab tool

### CODE REVIEW

File: galaxy/macro/macros.xml [hunk 1]
```diff
@@ -33,41 +33,50 @@
...
-                    <when value="single_file">
-                        <param name="single_file" type="data" format="mzxml,netcdf" label="Single file"  multiple="true"/>
-                    </when>
-                </conditional>
-            </when>
-        </conditional>
+    <xml name="input_file_load">
+        <section name="file_load_section" title="Resubmit your raw dataset or your zip file">
+            <conditional name="file_load_conditional">
+                <param name="file_load_select" type="select" label="Resubmit your dataset or your zip file" help="Use only if you get a message which say that your original dataset or zip file have been deleted on the server." >
```
Comment (bgruening) [inline galaxy/macro/macros.xml]: Resubmit your dataset? What does this mean? Can we make the yes can selected by default?

File: src/xcms_w4m_script/xcms.r [hunk 2]
```diff
@@ -223,8 +170,8 @@ if (thefunction  == "xcmsSet") {
 
     #transform the files absolute pathways into relative pathways
     xset@filepaths<-sub(paste(getwd(),"/",sep="") ,"", xset@filepaths)
-
-    if(exists("zipfile") && (zipfile!="")) {
+    print(zipfile)
```
Comment (bgruening) [inline src/xcms_w4m_script/xcms.r]: leftover debug output?

Comment (lecorguille) [inline src/xcms_w4m_script/xcms.r]: Oups

File: galaxy/macro/macros.xml (hunk 1 above)
Comment (lecorguille) [inline galaxy/macro/macros.xml]: Hehe!
One of the funny things about this package. 
Some steps of XCMS need to access the raw data files. 
So basically, we store the path of the input in the first step. 
But you can't do that for the functional test. So I introduced this "resubmit param
I tried to hidden it but it was buggy. I should retest this option

**Pull request merged**